src/lib/Sympa/Request/Handler/move_list.pm

```diff
@@ -392,7 +392,7 @@ sub _move {
     # Try renaming archive.
     # Continue even if there are some troubles.
     my $arc_dir     = $current_list->get_archive_dir;
-    my $new_arc_dir = $fake_list->get_archive_dir;
+    my $new_arc_dir = lc($fake_list->get_archive_dir);
```

Comment:
Is this fix means `$fake_list` contains the list name with uppercase letter? I'll investigate it for a while.

Reply:
Yes, it means that.

Reply:
Making line 187 in beginning of `_move()` to be
```perl
    my $listname     = lc($request->{listname} || );
```
looks better.

Reply:
Also line 459.

Reply:
I applied your suggestion and rewrote my commit (no need to keep my old solution in history).